build.gradle

```diff
@@ -21,6 +21,62 @@ allprojects {
     }
 }
 
+ext.versions = [
+        hadoop: '2.6.1',
+        hive  : '1.1.0',
+        restli: '1.15.7',
+        slf4j : '1.7.18',
+]
+
+ext.deps = [
```

Comment:
In my standard practice experience, dependencies are usually put in json file, like azkaban-web-server/package.json